qutebrowser/browser/history.py

```diff
@@ -208,4 +212,3 @@ def init(parent=None):
     """
     history = WebHistory(parent)
     objreg.register('web-history', history)
-    QWebHistoryInterface.setDefaultInterface(history)
```

Comment:
This will still be needed to color visited links. I suggest you re-add this, simply ignore what's happening in `addHistoryEntry`, and add another method (say `add_url` or so).